e2e/start-server

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -eo pipefail
```

Comment:
This is just to make the script fail on first error, to help debugging and increase robustness.